src/test/kotlin/jces1209/SlowAndMeaningful.kt

```diff
@@ -32,4 +30,18 @@ class SlowAndMeaningful private constructor(
         .skipSetup(true)
         .seed(12345L)
         .build()
+
+    class Builder {
+        private var browser: Class<out Browser> = EagerChromeBrowser::class.java
+        private var region: Regions = Regions.US_EAST_1
+
+        fun region(region: Regions) = apply { this.region = region }
```

Comment:
Shouldn't we have one for browser, too?

Reply:
You're right. We did need it already and otherwise `private var browser` would be pointless